### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by shubhwebkul**
### 1. Why is this change necessary?
Whenever a new issue will be raised, It'll be categorized on the basis of the issue template and some important details will be asked.

### 2. What does this change do, exactly?
Because of this change user will be asked to provide some important details before raising any issue.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Document_issue.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: ⛔ Documentation Issue
+about: See https://github.com/shopware/devdocs/issues for documentation issues
```
Comment (mitelg) [inline .github/ISSUE_TEMPLATE/Document_issue.md]: this is wrong for Shopware 6. the documentation is inside this repository :wink: 

devdocs are Shopware 5 only

Comment (shubhwebkul) [reply .github/ISSUE_TEMPLATE/Document_issue.md]: Hello @mitelg,
So if the document related issues will also be raised in the same repository, then i think there is no need for this file. I've updated the PR.

Thanks!

**Comment by pantrtxp** [comment]
Hi @shubhwebkul,

could you please squash your commits?

sunny regards
Pantrtxp

**Comment by shubhwebkul** [comment]
Hey @pantrtxp,
Done.

**Comment by mitelg** [comment]
hey @shubhwebkul 
thanks for your contribution :+1: :tada: :blue_heart: 
I imported your PR and it will be merged soon

**Comment by mitelg** [comment]
looks really nice https://github.com/shopware/platform/issues/new/choose
thanks again @shubhwebkul !

**Comment by shubhwebkul** [comment]
Thanku for merging my PR.
Will try to contribute more in shopware with full of my :heart:

**Pull request closed without merging**